src/java.base/share/classes/java/lang/ModuleLayer.java

```diff
@@ -881,6 +881,22 @@ public Optional<Module> findModule(String name) {
                 .findAny();
     }
 
+    /**
+     * Update module named {@code name} to allow access to restricted methods.
+     *
+     * @param name the name of the module for which the native access should be enabled
+     * @return {@code true} iff the module was present in this layer,
+     *         {@code false} otherwise
+     */
+    boolean addEnableNativeAccess(String name) {
```

Comment:
Do you mind changing the method description to "Updates the module with the given name in this layer to allow access to restricted methods"? This will be keep it more consistent with the exiting methods.

Also "was present" in the return description hints that it may not now be present. A module layer is immutable so it can just say that it returns true if the  module is in this layer.

Reply:
Adjusted here:
https://github.com/openjdk/jdk/pull/18106/commits/6af399ee4a3e908cb7c6b983b9747310e23a888e
please let me know if further/other changes/adjustment are desirable.

Thanks.